validation_components/validation.py

```diff
@@ -16,12 +16,13 @@ def validation_runner(arguments: argparse.Namespace):
         elif manifest_entry.query_id in registered_values:
             pass
         else:
-            timestamp = NcbiQuery.generate_new_timestamp(timestamp)
-            error_code, ncbi_common_name = manifest_entry.NcbiQuery.query_ncbi(manifest_entry)
+            connecter = NcbiQuery()
```

Comment:
Suggest to move the above line outside the loop, otherwise you create a new object every time. Once you do this, you can establish the "timestamp" as attribute of your NcbiQuery object, and you can "hide away" this whole process from the calling function.